crates/astria-sequencer/src/bridge/state_ext.rs

```diff
@@ -1182,7 +1016,7 @@ mod test {
 
         // uninitialized delete ok
         state
-            .clear_block_deposits()
+            .clear_block_deposit_nonces()
             .await
             .expect("calls to clear block deposit should succeed");
```

Comment:
```suggestion
            .expect("calls to clear block deposit nonces should succeed");
```

Reply:
Fixed in [803b3b4](https://github.com/astriaorg/astria/pull/1525/commits/803b3b4e413de94fbc471e177a3e2cfff96a52f9).